media/js/firefox/new/common/thanks-direct.es6.js

```diff
@@ -31,10 +31,25 @@
                 // Make sure the 'Try downloading again' link is well formatted! (issue 9615)
                 if (directDownloadLink && directDownloadLink.href) {
                     directDownloadLink.href = downloadURL;
+                    directDownloadLink.addEventListener(
+                        'click',
+                        function (event) {
+                            try {
+                                TrackProductDownload.handleLink(event);
+                            } catch (error) {
+                                return;
+                            }
+                        }
+                    );
```

Comment:
I think you can simplify this a bit by using an arrow function

(also make sure to add the `useCapture` param at the end for legacy browser support?):

```suggestion
                    directDownloadLink.addEventListener('click', (event) => {
                        try {
                            TrackProductDownload.handleLink(event);
                        } catch (error) {
                            return;
                        }
                    }, false);
```